### PR TITLE
RC-v1.8: reflect contract changes

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):

preseve existing invest secions UI to be rewired

## Explanation of the solution
NOTE: most of these changes can be seen in `src/types/contracts` - everything else are adjustment 
* remove strategy editor
* update `type EndowmentDetails` and consumers
* remove endow list query and consumers 
* remove unused Accounts methods
* remove registrar vault rates query and types
* remove registrar vault list query and types

### on separate PR
- update invest and redeem types and wiring
- update source of list of strategies

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Admin/Charity/Account/Balances.tsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-import { useAssetsQuery } from "services/juno/custom";
 import ContentLoader from "components/ContentLoader";
 import QueryLoader from "components/QueryLoader";
 import { humanize } from "helpers";
 
 export default function Balances({ type }: { type: AccountType }) {
   const { id } = useAdminResources();
-  const { data, ...rest } = useAssetsQuery({ endowId: id });
-
+  console.log({ id, type });
```
Comment (misicnenad) [inline src/pages/Admin/Charity/Account/Balances.tsx:9]:
```suggestion
```

File: src/pages/Admin/Charity/Account/Positions.tsx [hunk 2]
```diff
@@ -10,15 +9,11 @@ type Props = {
 
 export default function Positions({ type }: Props) {
   const { id } = useAdminResources();
-  const { data, ...rest } = useAssetsQuery({
-    endowId: id,
-  });
-
+  console.log({ type, id });
```
Comment (misicnenad) [inline src/pages/Admin/Charity/Account/Positions.tsx:12]:
```suggestion
```

File: src/pages/Admin/Charity/Invest/Invesments.tsx [hunk 3]
```diff
@@ -10,13 +9,16 @@ type Props = {
 
 export default function Investments({ type }: Props) {
   const { id } = useAdminResources();
-  const { data, ...rest } = useAssetsQuery({
-    endowId: id,
-  });
+  console.log({ id, type });
```
Comment (misicnenad) [inline src/pages/Admin/Charity/Invest/Invesments.tsx:12]:
```suggestion
```

File: src/pages/Admin/Charity/common/Balances/Balance.tsx [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
 import { useAdminResources } from "pages/Admin/Guard";
-import { useAssetsQuery } from "services/juno/custom";
 import QueryLoader from "components/QueryLoader";
 import { humanize } from "helpers";
 
 type Props = { type: AccountType };
 export default function Balance({ type }: Props) {
   const { id } = useAdminResources();
-  const { data, ...rest } = useAssetsQuery({ endowId: id });
+  console.log({ id });
```
Comment (misicnenad) [inline src/pages/Admin/Charity/common/Balances/Balance.tsx:10]:
```suggestion
```

**Review by misicnenad — APPROVED** [review]

**Review by misicnenad — CHANGES_REQUESTED** [review]
Leftover console logs.

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**